readers/hjson.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const fs = require('fs');
+const Hjson = require('hjson');
+
+exports.load = function (name) {
+    return Hjson.parse(fs.readFileSync(name, "utf8"));
```

Comment:
Oh, just noticed this: per the JS conventions, the case for libraries is always lower case, except for constructors.

Reply:
I was going to use it first, but then I went with what hjson's npm description gives as example. Shall be fixed.

Reply:
See: https://stackoverflow.com/questions/1564398/javascript-method-naming-lowercase-vs-uppercase  (we deviate from the conventions slightly with the use of under_score names instead of camelCase, but we're pretty consistent otherwise.)

Reply:
Ok, I will remember that.